integration/tests/inferentia/inferentia_test.go

```diff
@@ -98,38 +100,15 @@ var _ = Describe("(Integration) Inferentia nodes", func() {
 				)
 				Expect(cmd).To(RunSuccessfully())
 			})
-			It("should have installed the neuron device plugin", func() {
-				cfg := &api.ClusterConfig{
```

Comment:
I'm just gonna have to smash the copy paste key into little pieces :shrug: only way to break the habit

Reply:
hahaha